action_topic: "/game_action"
behavior_topic: "/steering_behavior"
vel_topic_pub: "/game_navigation/unsafe/cmd_vel"
vel_topic_sub: "/vel"
player_angle_topic: "/player_filtered"
tower_pos_topic: "/tower_position"
safety_topic: "/safety"
scan_topic: "/scan"
tower_rectangle_topic: "/tower_rectangle"
deception_topic: "/deception"

number_sections_safety: 8
safety_treshold: 0.64